mta/entities/__init__.py

```diff
@@ -48,14 +49,24 @@ class BaseLoggedInPage(View):
     # only if no project available
     blank_state = View.nested(BlankStateView)
 
+    def get_url(self):
```

Comment:
Optional: it not returning something but `bool` value. It will be great if we use a team like `validate_url`

Reply:
Done